src/caselawclient/Client.py

```diff
@@ -331,6 +333,22 @@ def checkin_judgment(self, judgment_uri: str) -> requests.Response:
             accept_header="application/xml",
         )
 
+    def get_judgment_checkout_status(self, judgment_uri: str) -> str:
+        uri = f"/{judgment_uri.lstrip('/')}.xml"
+        xquery_path = os.path.join(
+            ROOT_DIR, "xquery", "get_judgment_checkout_status.xqy"
+        )
+        vars = {
+            "uri": uri
+        }
+
+        response = self.eval(
+            xquery_path,
+            vars=json.dumps(vars),
+            accept_header="application/xml",
+        )
+        return self.parse_checkout_status_response(response.text)
```

Comment:
This function feels like an outlier, in that it doesn't return an `eval()`d response, but a parsed string.

There might be value in extracting a common "call remote XQuery" function but equally we might not be ready for that.